Dockerfile

```diff
@@ -33,7 +16,21 @@ COPY emails emails
 ENV NODE_ENV production
 RUN ./node_modules/.bin/grunt build
 
-# Final container
+FROM golang:1.14.1-alpine AS go-builder
```

Comment:
These changes seems unrelated to these changes? Can you open a separate PR for this?

Reply:
@marefr I can.